field_formatter.module

```diff
@@ -14,4 +14,5 @@ function field_formatter_field_formatter_info_alter(array &$info) {
   $field_types = array_keys($field_type_manager->getDefinitions());
 
   $info['field_link']['field_types'] = $field_types;
+  $info['field_formatter_limite']['field_types'] = $field_types;
```

Comment:
Ups you are right.

As always I should have written some tests before :)

On Sun, Jul 10, 2016 at 2:30 PM, marcvangend <notifications@github.com>
wrote:

> In field_formatter.module
> <https://github.com/drupal-media/field_formatter/pull/5#discussion_r70181179>
> :
>
> > @@ -14,4 +14,5 @@ function field_formatter_field_formatter_info_alter(array &$info) {
> >    $field_types = array_keys($field_type_manager->getDefinitions());
> >
> >    $info['field_link']['field_types'] = $field_types;
> > +  $info['field_formatter_limite']['field_types'] = $field_types;
>
> "limite" is a typo I guess
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/drupal-media/field_formatter/pull/5/files/ada2ec3a0bfcdc06ba9dbbccba586a64e557bfa2#r70181179>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz7oxlvRFsvyPm5fy6D58acyoSPjD3ks5qUOX5gaJpZM4JIyGi>
> .
>